### DISCUSSION TITLE
Connect wallet with Web3auth Flutter SDK

### DISCUSSION THREAD

**Discussion by nisa10880** [Web3Auth/Web3Auth#740] [Help]
Hello,

I am a little bit confused about how can we connect to a wallet with the flutter SDK. Is a wallet created after login with a provider ? And how to use and connect to this wallet if it's created ?

**Comment by nisa10880** [comment]
You can use web3dart to connect to a wallet and using the private from web3auth response.

       
```
              final Web3AuthResponse response = await Web3AuthFlutter.login(
                 LoginParams(
                   loginProvider: w3enums.Provider.email_passwordless,
                   redirectUrl: redirectUrl,
                 ),
               );
               Credentials fromHex = EthPrivateKey.fromHex(response.privKey);
     
              var address = await fromHex.extractAddress();
```

**Comment by Bravark** [comment]
Does this give the app access to the users private key..?
